ember.md

```diff
@@ -46,6 +51,7 @@ CSS is permitted (and encouraged) in apps and addons under certain circumstances
   background: $color1;
 
   .chatMsg {
+	font-family: $font1,
```

Comment:
the bad comment on line 47 is different from the master branch. it should match master. https://github.com/ASHdevelopment/standards/blob/master/ember.md#css